cdap-api/src/main/java/co/cask/cdap/api/app/ApplicationConfigurer.java

```diff
@@ -122,4 +123,11 @@ ScheduleBuilder buildSchedule(String scheduleName, ProgramType programType,
    * @param scheduleCreationSpec defines the schedule.
    */
   void schedule(ScheduleCreationSpec scheduleCreationSpec);
+
+  /**
+   * Get a TriggerFactory to get triggers.
+   *
+   * @return The {@link TriggerFactory} used to get triggers
+   */
+  TriggerFactory getTriggerFactory();
```

Comment:
it's enough for it to return the same instance for every call, so I think it's a get